### PR TITLE
Jauregui elsol#1

### LINKED ISSUE #6: Create a new file for david.i.jauregui Granada Hills, CA
- [x] Create a branch
- [x] Add a commit
- [x] Open a PR
- [ ] Add a discussion
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by davidjauregui**
closes #6

### CODE REVIEW

**Comment by ReneButcher** [comment]
I generally comment on your help wanted with "HEY" :+1:

File: jauregui-granadahills.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Granada Hills, CA
+
+### Places to Eat
+My house
+Asahi Sushi
+Chipotle
```
Comment (ReneButcher) [inline jauregui-granadahills.md:6]: I'm afraid of Chipotle....bad bacteria

Comment (davidjauregui) [reply jauregui-granadahills.md:6]: Its not a place I would eat

File: jauregui-granadahills.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Granada Hills, CA
+
+### Places to Eat
+My house
```
Comment (ReneButcher) [inline jauregui-granadahills.md:4]: When can I come to your house for FOOD?

File: jauregui-granadahills.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Asahi Sushi
+Chipotle
+Denny's
+Del Taco (Rene's favorite)
+
+### Places to Go
+Lowes
+Home Depot
```
Comment (ReneButcher) [inline jauregui-granadahills.md:12]: I added a comment above and used the "start a review" vs "add a 1 line comment"

**Review by ReneButcher — COMMENTED** [review]
now there is a button to finish the review (DONE)

**Review by ReneButcher — APPROVED** [review]

**Pull request merged**